### ISSUE TITLE
Better communication thru Discord Chat

### ISSUE THREAD

**Issue by donPuerto** [Lolliedieb/lolMiner-releases#1124]
Do we have discord Chat?

**Comment by jgonzis** [comment]
> Do we have discord Chat?

There is a discord and Telegram group... normally Telegram group is more active 

Telegram : https://t.me/lolMiner
Discord: https://discord.gg/ywDNe9dR

**Comment by sxemini** [comment]
Yes because this discord is not working and nobody knows that this discord exists. As i said in the telegram group 1 month ago or so, bad public work result in less user base. After ton mining the user base will decrease fast again.

**Comment by jgonzis** [comment]
> Yes because this discord is not working and nobody knows that this discord exists. As i said in the telegram group 1 month ago or so, bad public work result in less user base. After ton mining the user base will decrease fast again.

The discord hasn't be done publicity. The telegram is the way that the Dev has been using, but Discord was created 1 year ago, but not used... Let's hope now more people use it. 
It could end 1st ETH than TON haha... let's see what happens in June

**Comment by nicjohnston** [comment]
The discord link above is invalid, has the discord been closed or could someone post a new invite link?

**Comment by jgonzis** [comment]
> The discord link above is invalid, has the discord been closed or could someone post a new invite link?

https://discord.gg/jvfRvK5wTv
